### DISCUSSION TITLE
Moving plots to a new machine - how to?

### DISCUSSION THREAD

**Discussion by chiachain** [Chia-Network/chia-blockchain#1641] [Q&A]
I plugged in my hdd with the plots on them to the new computer, and imported my key.  The plots show in the "Not found plots" section. There is the Do you have existing plots on this machine? Add Plot Directory"" but I can only select the drive where the plots are and nothing happens when I just select the drive.  Am I supposed to create a plot directory, before I add it? How would I do that? Thanks much.

**Comment by hoffmang9** [comment]
Your drive is probably mounted with root ownership such that your user account can't actually read the directories or files.

**Reply by chiachain** [reply]
Thanks for the reply. I am running Windows 10 and using the GUI.  I just changed drive letters, and it worked.
